testing/it/ui-js/src/content/jcr_root/apps/core/wcm/sandbox/tests/test-cases/Tabs/v1/Tabs.js

```diff
@@ -114,30 +114,44 @@
     /**
      * Create child items
      */
-    tabs.tcCreateItems = function(selectors) {
+    tabs.tcCreateItems = function(selectors, component, cmpPath) {
         return new h.TestCase("Create child items")
             // open the edit dialog
-            .execTestCase(c.tcOpenConfigureDialog("cmpPath"))
+            .execTestCase(c.tcOpenConfigureDialog(cmpPath))
             .click(selectors.editDialog.childrenEditor.addButton)
             .wait(200)
-            .click(selectors.insertComponentDialog.components.responsiveGrid)
+            .click(component)
             .wait(200)
             .fillInput(selectors.editDialog.childrenEditor.item.last + " " + selectors.editDialog.childrenEditor.item.input, "item0")
             .click(selectors.editDialog.childrenEditor.addButton)
             .wait(200)
-            .click(selectors.insertComponentDialog.components.responsiveGrid)
+            .click(component)
             .wait(200)
             .fillInput(selectors.editDialog.childrenEditor.item.last + " " + selectors.editDialog.childrenEditor.item.input, "item1")
             .click(selectors.editDialog.childrenEditor.addButton)
             .wait(200)
-            .click(selectors.insertComponentDialog.components.responsiveGrid)
+            .click(component)
             .wait(200)
             .fillInput(selectors.editDialog.childrenEditor.item.last + " " + selectors.editDialog.childrenEditor.item.input, "item2")
             // save the edit dialog
             .execTestCase(c.tcSaveConfigureDialog)
             .wait(200);
     };
 
+    /**
+     * Create tab items
+     */
+    tabs.tcAddTabs = function(selectors, component, parentPath, cmpPath, tabName) {
+        return new h.TestCase("Create child items")
```

Comment:
@andreeadracea - this description needs updating. Also are we really creating multiple tabs here? Looks like you only add one tab and name it? so I would rename this `tcAddTab`